### PR TITLE
Docker Compose V2 and related build fixes

### PR DESCRIPTION
**Pull request by gramound**
Misc build fixes discussed in b/245490832

1. Replacing multiple YAML '<<:' keys with array
2. Replacing 'python' with 'python3'
3. Removing duplicate 'image:' key in xvfb (low confidence, I don't have access to test it)
4. Replacing 'docker-compose' commands with 'docker compose' (with space)

### CODE REVIEW

File: docker-compose-windows-internal.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 The Cobalt Authors. All Rights Reserved.
```
Comment (andrewsavage1) [inline docker-compose-windows-internal.yml:1]: The copyright year represents when the file was created, we don't want to change it

**Review by andrewsavage1 — CHANGES_REQUESTED** [review]
I'm unsure about the yaml merge key changes, will leave Oscar or Kaido to review that

**Comment by gramound** [comment]
Ah yes, sorry about the copyright dates. The `check-copyright-year` hook was asking me to change dates and add copyright headers to ~300 files. I later realized it was because my changes were on top of the googlesource.com main branch (no common history with github, so it must have thought I was adding new files). I'll undo those date changes in my next submission.

**Review by oxve — APPROVED** [review]

**Review by andrewsavage1 — APPROVED** [review]

**Comment by kaidokert** [comment]
Thank you very much for your contributions ! Really helpful. We'll ensure everything is stable and working and will consider for 24lts as well.

**Pull request merged**